src/zcl/definition/cluster.ts

```diff
@@ -3541,6 +3541,14 @@ const Cluster: {
         commandsResponse: {
         }
     },
+    draytonDeviceInfo: {
```

Comment:
Maybe rename this to `wiserDeviceInfo`, my test unit branded as 'Eberle Wiser' also reports with the exact same database entry.

I looks like both Drayton and Eberle are local rebrands from the Wiser device (which seems to be made by Schnieder)